smartsim/experiment.py

```diff
@@ -166,12 +166,18 @@ def start(self, *args, block=True, summary=False):
         :param summary: print a launch summary prior to launch,
                         defaults to False
         :type summary: bool, optional
+        :param kill_on_interrupt: flag for killing jobs when SIGINT is received
```

Comment:
To clarify my own understanding, if `kill_on_interrupt` is `False`, do we exit the smartsim process without cleaning up the jobs? 

It might be helpful to clarify the behavior for True/False in this docstring (and in the previous function).